webpack.config.ts

```diff
@@ -11,28 +11,63 @@ import CopyWebpackPlugin from 'copy-webpack-plugin';
 import MiniCssExtractPlugin from 'mini-css-extract-plugin';
 import webpack, {Configuration} from 'webpack';
 
-function parseFeatureDetails(id: FeatureID): FeatureMeta {
-	const content = readFileSync(`source/features/${id}.tsx`, {encoding: 'utf-8'});
-	const fields = ['disabled', 'description', 'screenshot'] as const;
+const readmeContent = readFileSync('readme.md', 'utf-8');
+
+function stripLinks(markdownText: string): Partial<FeatureMeta> {
+	const urls: string[] = [];
+	const description = markdownText.replace(/\[(.+?)]\((.+?)\)/g, (_match, title, url) => {
+		urls.push(url);
+		return title;
+	});
+
+	return {
+		description,
+		screenshot: urls.find(url => url.startsWith('https://user-images.githubusercontent.com/')) ?? urls[0]
+	};
+}
 
+function parseFeatureDetails(id: FeatureID): FeatureMeta {
 	const feature: Partial<FeatureMeta> = {id};
-	for (const field of fields) {
-		const value = new RegExp(`\n\t${field}: '([^\\n]+)'`).exec(content)?.[1];
-		if (value) {
-			const validValue = value.trim().replace(/\\'/g, '’'); // Catch trailing spaces and incorrect apostrophes
-			if (value !== validValue) {
-				throw new Error(stripIndent(`
-					❌ Invalid characters found in \`${id}\`. Apply this patch:
-
-					- ${field}: '${value}'
-					+ ${field}: '${validValue}'
-				`));
-			}
 
-			feature[field] = value.replace(/\\\\/g, '\\');
+	const lineRegex = new RegExp(`^- \\[\\]\\(# "${id}"\\)(?: 🔥)? (.+)$`, 'm');
+	const lineMatch = lineRegex.exec(readmeContent);
+	if (lineMatch) {
+		Object.assign(feature, stripLinks(lineMatch[1]));
+		feature.description = feature.description!
+			.replace(/<code>\\`(.+?)\\`<\/code>/, '`$1`') // Simplify weird Markdown escaping
+			.replace(/<kbd>(.+?)<\/kbd>/g, '`$1`'); // Replace keyboard shortcut tags
+	} else {
+		// Feature might be highlighted in the readme
+		const imageRegex = new RegExp(`<img id="${id}" alt="(.+?)" src="(.+?)">`);
```

Comment:
This duplicates the content in the table again.

Let's simplify the table altogether. Use this markup, no changes required (other than the content)

```html
<table>
	<tr>
		<th width="50%">
			<p><a title="show-whitespace"></a> Makes whitespace characters visible
			<p><img src="https://user-images.githubusercontent.com/1402241/61187598-f9118380-a6a5-11e9-985a-990a7f798805.png">
		<th width="50%">
			<p>Adds one-click merge conflict fixers
			<p><img src="https://user-images.githubusercontent.com/1402241/54978791-45906080-4fdc-11e9-8fe1-45374f8ff636.png">
	<tr>
		<th>
			<p><a title="show-whitespace"></a> Makes whitespace characters visible
			<p><img src="https://user-images.githubusercontent.com/1402241/61187598-f9118380-a6a5-11e9-985a-990a7f798805.png">
		<th>
			<p>Adds one-click merge conflict fixers
			<p><img src="https://user-images.githubusercontent.com/1402241/54978791-45906080-4fdc-11e9-8fe1-45374f8ff636.png">
	<tr>
		<th>
			<p><a title="show-whitespace"></a> Makes whitespace characters visible
			<p><img src="https://user-images.githubusercontent.com/1402241/61187598-f9118380-a6a5-11e9-985a-990a7f798805.png">
		<th>
			<p>Adds one-click merge conflict fixers
			<p><img src="https://user-images.githubusercontent.com/1402241/54978791-45906080-4fdc-11e9-8fe1-45374f8ff636.png">
</table>
```

This example looks like:

<img width="734" alt="" src="https://user-images.githubusercontent.com/1402241/96837250-b8a5c700-140b-11eb-9b76-9be617273e2d.png">